packages/psss/src/components/Table/CellComponents/AlertMenuCell.js

```diff
@@ -2,12 +2,30 @@
  * Tupaia
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
-import React from 'react';
+import PropTypes from 'prop-types';
+import React, { useCallback, useContext } from 'react';
 import MoreVertIcon from '@material-ui/icons/MoreVert';
 import { IconButton } from '@material-ui/core';
+import { AlertsPanelContext } from '../../../containers';
 
-export const AlertMenuCell = () => (
-  <IconButton>
-    <MoreVertIcon />
-  </IconButton>
-);
+export const AlertMenuCell = React.memo(({ organisationUnit, period, syndromeName }) => {
+  const { setIsOpen, setData } = useContext(AlertsPanelContext);
+  const data = { organisationUnit, period, syndromeName };
+
+  const handleClick = useCallback(() => {
+    setData(data);
+    setIsOpen(true);
+  }, [data, setData, setIsOpen]);
+
+  return (
+    <IconButton onClick={handleClick}>
```

Comment:
I'm a bit confused here. In the design this button is for opening a menu with options to "Archive Alert" or "Create Outbreak". If that is no longer the plan then I think we should just remove this button but I don't think it is a good icon and placement for the opening of the side panel.

Reply:
Hey Tom, thanks for pointing this out! I indeed assumed that his button was used for opening the side panel, but I guess I got it wrong.

Perhaps there is a newer mockup but that's the one I am using: https://www.figma.com/file/N1AqYzfAXsTv4FHvAyb8a7/PSSS-Admin---Desktop?node-id=205%3A630 which is quite different from the existing components as far as alerts are regarded, so it often couldn't be used as a UI guide for alerts.

Let's have a chat so that you can explain me in more detail!